tasks/components/KanbanHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import TriangleDown from '../icons/TriangleDown'
+import { Input } from './Input'
+
+type KanbanHeaderTranslation = {
+  tasks: string,
+  status: string,
+  label: string
+}
+
+const defaultKanbanHeaderTranslations: Record<Languages, KanbanHeaderTranslation> = {
+  en: {
+    tasks: 'Tasks',
+    status: 'Status',
+    label: 'Label'
+  },
+  de: {
+    tasks: 'Aufgaben',
+    status: 'Status',
+    label: 'Label'
+  }
+}
+
+type KanbanHeaderProps = {
+  sortingStatus?: string,
+  sortingLabel?: string,
+  searchValue: string,
+  onSearchChange: (search: string) => void
+}
+
+export const KanbanHeader = ({
+  language,
+  sortingStatus,
+  sortingLabel,
```

Comment:
Both props are not in use

Reply:
because we haven't establish a sorting for the status or label? But the ui indicated the possibility

Reply:
I will just omit them for now in the function props